cpu/stm32/periph/i2c_2.c

```diff
@@ -39,16 +39,16 @@
 #include <errno.h>
 
 #include "cpu.h"
-#include "irq.h"
 #include "mutex.h"
 #include "pm_layered.h"
-#include "panic.h"
```

Comment:
`panic.h` seems to be used eg.: l.:516 `core_panic(PANIC_GENERAL_ERROR, "I2C FAULT");`